src/components/Lineage/Graph/index.tsx

```diff
@@ -154,7 +171,8 @@ export function EgeriaLineageGraph(props: Props) {
               debug={false}
               graphDirection={'HORIZONTAL'}
               selectedNodeId={guid}
-              actions={<EgeriaLineageGraphActions rawData={{...rawData}} printUri={printUri} />}
+              actions={<><EgeriaLineageGraphActions rawData={{...rawData}} printUri={printUri} />
+                         <Switch label="Processes" size="xs" disabled={selectedNodeGroup === 'Process'} checked={includeProcess} onChange={() => onSwitchChange(lineageType, !includeProcess)}/></>}
```

Comment:
i see that this is duplicated to all tabs which means you need to find a way of adding it to the `<EgeriaLineageGraphActons .../>` which aggregates all the added buttons and functionalities.